src/main/resources/db/changelog/changesets/changelog_20240124T152518Z.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="labidiayo (generated)" id="1706109930287-1">
+        <createTable tableName="load_flow_parameters">
+            <column name="id" type="UUID">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="load_flow_parametersPK"/>
+            </column>
+            <column name="balance_type" type="VARCHAR(255)"/>
+            <column name="connected_component_mode" type="VARCHAR(255)"/>
+            <column defaultValueBoolean="false" name="dc" type="BOOLEAN"/>
+            <column defaultValueNumeric="1.0" name="dc_power_factor" type="DOUBLE PRECISION"/>
+            <column defaultValueBoolean="true" name="dc_use_transformer_ratio" type="BOOLEAN"/>
+            <column defaultValueBoolean="true" name="distributed_slack" type="BOOLEAN"/>
+            <column defaultValueBoolean="true" name="hvdc_ac_emulation" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="phase_shifter_regulation_on" type="BOOLEAN"/>
+            <column defaultValueBoolean="true" name="read_slack_bus" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="shunt_compensator_voltage_control_on" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="transformer_voltage_control_on" type="BOOLEAN"/>
+            <column defaultValueBoolean="false" name="twt_split_shunt_admittance" type="BOOLEAN"/>
+            <column defaultValueBoolean="true" name="use_reactive_limits" type="BOOLEAN"/>
+            <column name="voltage_init_mode" type="VARCHAR(255)"/>
+            <column defaultValueBoolean="false" name="write_slack_bus" type="BOOLEAN"/>
```

Comment:
You use primitive `boolean` instead of object `Boolean` (same for double field) in your JPA entity, so it must be non-nullable because else `null` in database would be mapped as `false` in entity.
```suggestion
            <column defaultValueBoolean="false" name="dc" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueNumeric="1.0" name="dc_power_factor" type="DOUBLE PRECISION">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="true" name="dc_use_transformer_ratio" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="true" name="distributed_slack" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="true" name="hvdc_ac_emulation" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="false" name="phase_shifter_regulation_on" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="true" name="read_slack_bus" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="false" name="shunt_compensator_voltage_control_on" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="false" name="transformer_voltage_control_on" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="false" name="twt_split_shunt_admittance" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column defaultValueBoolean="true" name="use_reactive_limits" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
            <column name="voltage_init_mode" type="VARCHAR(255)"/>
            <column defaultValueBoolean="false" name="write_slack_bus" type="BOOLEAN">
                <constraints nullable="false"/>
            </column>
```